Sources/AppStoreConnectCLI/Model/API/Sales.Filter+ExpressibleByArgument.swift

```diff
@@ -9,7 +9,7 @@ private typealias Filter = DownloadSalesAndTrendsReports.Filter
 extension Filter.Frequency: ExpressibleByArgument, CustomStringConvertible {
     private typealias AllCases = [Filter.Frequency]
```

Comment:
You could probably fix this by changing:
```suggestion
    public typealias AllCases = [Filter.Frequency]
```